src/Fetch.ml

```diff
@@ -216,6 +218,14 @@ let decodeRequestRedirect = (* internal *)
   | "manual" -> Manual
   | e -> raise (Failure ("Unknown requestRedirect: " ^ e))
 
+module AbortController = struct
+  type t = abortController
+  
+  external getSignal : t -> signal = "signal" [@@bs.get]
+  external abort : t -> unit = "abort" [@@bs.send]
```

Comment:
Not that it matters all that much, but every method here uses `bs.send.pipe` instead of `bs.send`, and it'd be nice to be consistent.